core/src/i18n/en.yaml

```diff
@@ -2,18 +2,18 @@ addons:
   addons-status: Addons Status
   description: <0>Addons Configuration</0> defines Namespace-scoped addons that provide all information required to convert Helm charts into Service Classes in Service Catalog.
   errors:
-    cannot-create: Cannot create the Addons Configuration.
+    cannot-create: Cannot create the Addons Configuration
   headers:
     url: URL
   messages:
```

Comment:
In case of messages and tooltips, it's good to use sentence case. If there are full sentences, let's use punctuation. Contrary wise, if the message is only a clause (fragment of a sentence), punctuation feels odd and redundant.